test/regression/expected/test_all_types.out

```diff
@@ -0,0 +1,129 @@
+\x on
+SET bytea_output = 'escape';
+SELECT * FROM duckdb.query($$
+FROM test_all_types()
+SELECT * exclude(
+    varint,
+    TIME,
+    time_tz,
+    bit,
+    small_enum,
+    medium_enum,
+    large_enum,
+    timestamptz_array,
+    struct,
+    struct_of_arrays,
+    array_of_structs,
+    map,
+    "union",
+    fixed_int_array,
+    fixed_varchar_array,
+    fixed_nested_int_array,
+    fixed_nested_varchar_array,
+    fixed_struct_array,
+    struct_of_fixed_array,
+    fixed_array_of_int_list,
+    list_of_fixed_int_array,
+    nested_int_array, -- The nested array has different lengths, which is not possible in PG
+)
+$$)
+-[ RECORD 1 ]---+-------------------------------------------------------------------------------------------------------------------------------------
+bool            | f
+tinyint         | \200
+smallint        | -32768
+int             | -2147483648
+bigint          | -9223372036854775808
+hugeint         | 0
+uhugeint        | 0
+utinyint        | 0
+usmallint       | 0
+uint            | 0
+ubigint         | 0
+date            | 07-14-5881580
+timestamp       | Sun Jan 10 08:01:49.551616 294247
+timestamp_s     | Sun Jan 10 08:01:49.551616 294247
+timestamp_ms    | Sun Jan 10 08:01:49.551616 294247
+timestamp_ns    | Wed Sep 22 00:00:00 1677
+timestamp_tz    | Sun Jan 10 00:01:49.551616 294247 PST
```

Comment:
All of these overflow currently. I think we should either truncate or throw an error in this case.